http.go

```diff
@@ -276,7 +276,11 @@ func httpPutReputation(w http.ResponseWriter, r *http.Request) {
 	}
 	exc := false
 	if rep.Type == "ip" {
-		exc = isException(rep.Object)
+		if exc, err = isException(rep.Object); err != nil {
```

Comment:
My only concern with this approach is that we are treating `isException()` as an IP validator as well. It might be worth splitting this up into an IP validation step and an is this an exception step.

For IP validation, you can do something like this: https://github.com/mozilla-services/foxsec-pipeline-contrib/blob/master/slackbot-background/utils.go#L45-L46

Reply:
Since this is just for logging, should just log the warning and not return a `500`